flow360/component/flow360_params/flow360_params.py

```diff
@@ -1076,6 +1096,8 @@ def _init_no_context(self, filename, legacy_fallback=False, **kwargs):
             with UnitSystem.from_dict(**unit_system):
                 super().__init__(**model_dict)
         else:
+            ## Legacy version always uses flow360 unit
+            unit_system_manager.set_current(flow360_unit_system)
```

Comment:
this is not right. The legacy updater should decide. The problem is with "comments" section produces by webUI which will use SI unit system. Also, `set_current` should not be used, because it goes around lock. It probably should be renamed to `_set_current`

Reply:
Sure. Removed.